PP-899 - get rid of `signup/credentials` (#429)
livechat/livechat-public-docs: site/content/sign-in-with-livechat/sdk-documentation.md

* **linkLabel** – (optional, defaults to empty string) allows to change default _Sign in with LiveChat_ label with custom text.

> * **popupRoute** – (optional, defaults to empty string) it lets you change default popup's endpoint, possible values are:<br><br> • `''` (default) – empty string which will show sign-in view.<br><br> • `'signup'` – it will point user directly to the first step of sign-up.

Reviewer: ```suggestion
* **popupRoute** – (optional, default: an empty string) it lets you change the default popup endpoint. Possible values are:<br><br> • `''` (default) – an empty string, which will show the sign-in view.<br><br> • `'signup'` – it will point a user directly to the first step of the signup.
```

Author: https://www.lexico.com/en/definition/default

Author: done in ab3abc136c942da13cc5a8a055b4f8ef792848f7
